slack-util.el

```diff
@@ -55,5 +55,24 @@
                                    nil
                                  value)))))))
 
+(eval-after-load "company"
+  '(progn
+     (defun company-slack-backend (command &optional arg &rest ignored)
+       "Completion backend for slack chats.  It currently understands
+@USER; adding #CHANNEL should be a simple matter of programming."
+       (interactive (list 'interactive))
+       (cl-case command
+         (interactive (company-begin-backend 'company-slack-backend))
+         (prefix (when (and (eq major-mode 'slack-mode) (looking-back "\\W@\\(\\w*\\)")) (match-string 1)))
+         (candidates (remove-if-not
+                      (lambda (x)
+                        (s-starts-with-p arg x))
```

Comment:
there is no dependency to `s.el` in this package. please replace `s-starts-with-p` with `string-prefix-p`.